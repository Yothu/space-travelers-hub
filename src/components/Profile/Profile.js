const Profile = () => (
  <div>
    <div>
      <p>profile</p>
    </div>
  </div>
);

export default Profile;
