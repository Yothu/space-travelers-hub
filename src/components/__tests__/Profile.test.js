import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import Profile from '../Profile/Profile';
import store from '../../redux/configureStore';

describe('Profile page', () => {
  it('it renders', () => {
    const profile = renderer
      .create(
        <Provider store={store}>
          <Profile />
        </Provider>,
      )
      .toJSON();
    expect(profile).toMatchSnapshot();
  });
});
