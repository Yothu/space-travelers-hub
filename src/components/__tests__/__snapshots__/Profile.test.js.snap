// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile page it renders 1`] = `
<div
  className="sc-bdvvtL jaFYCD"
>
  <div
    className="sc-gsDKAQ eEigVt"
  >
    <div
      className="sc-hKwDye dTwmPF font-1"
    >
      My Rockets
    </div>
    <div
      className="sc-eCImPb jgFGSY"
    />
  </div>
  <div
    className="sc-dkPtRN UWZAf"
  >
    <div
      className="sc-hKwDye dTwmPF font-1"
    >
      My Missions
    </div>
    <div
      className="sc-jRQBWg jFjSsb"
    />
  </div>
</div>
`;
